src/providers/gift-card/gift-card.ts

```diff
@@ -98,8 +98,7 @@ export class GiftCardProvider extends InvoiceProvider {
       amount: data.amount,
       clientId: data.uuid,
       discounts: data.discounts,
-      email: data.email,
-      transactionCurrency: data.buyerSelectedTransactionCurrency
+      email: data.email
```

Comment:
`selectPaymentOption` sets the `buyerSelectedTransactionCurrency` so will throw an error if `buyerSelectedTransactionCurrency` is already preset.